plugins/google-sheets/src/pages/MapSheetFields.tsx

```diff
@@ -148,28 +146,50 @@ export function MapSheetFieldsPage({
 }: Props) {
     const { ref: scrollRef, inView: isAtBottom } = useInView({ threshold: 1 })
 
+    const uniqueColumnNames = useMemo(() => generateUniqueNames(headerRow), [headerRow])
```

Comment:
Maybe this could be stored in `pluginContext`?

Reply:
Nah, can't, due to https://github.com/framer/company/issues/30923. This PR actually fixes that.